src/JetstreamServiceProvider.php

```diff
@@ -160,12 +164,14 @@ protected function configurePublishing()
      */
     protected function configureRoutes()
     {
-        Route::group([
-            'namespace' => 'Laravel\Jetstream\Http\Controllers',
-            'domain' => config('jetstream.domain', null),
-        ], function () {
-            $this->loadRoutesFrom(__DIR__.'/../routes/'.config('jetstream.stack').'.php');
-        });
+        if (Jetstream::$registersRoutes) {
```

Comment:
@m1guelpf Could you add a matching method to the Jetstream class and call that here, instead of directly accessing the class property? It would keep the code consistent with the patterns used across this package